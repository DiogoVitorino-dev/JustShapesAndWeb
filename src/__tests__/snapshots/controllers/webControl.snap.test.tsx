/**
 * @jest-environment jsdom
 */

import { render } from "@testing-library/react-native";

import WebControl from "@/controllers/web";

describe("Web Control - Snapshot web control test", () => {
  it("should render correctly", () => {
    const tree = render(
      <WebControl velocity={100} onMove={() => {}} />,
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
