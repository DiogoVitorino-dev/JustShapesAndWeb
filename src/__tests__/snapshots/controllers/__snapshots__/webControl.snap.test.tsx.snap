// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Web Control - Snapshot web control test should render correctly 1`] = `
<RNCSafeAreaProvider
  onInsetsChange={[Function]}
  style={
    [
      {
        "flex": 1,
      },
      undefined,
    ]
  }
/>
`;
